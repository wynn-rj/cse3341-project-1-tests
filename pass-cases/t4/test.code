		program 
 int	
x ,y, z;
begin input x,y;z:=x;x :=y;y:=
z;output
x;output 
y ;	end 	 
