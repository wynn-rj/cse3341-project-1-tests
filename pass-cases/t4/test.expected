program
  int x,y,z;
begin
  input x,y;
  z:=x;
  x:=y;
  y:=z;
  output x;
  output y;
end
5
3
